modules/log6x-6-1-5-rn.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.1/log6x-release-notes-6.1.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-6-1-5_{context}"]
+= Logging 6.1.5 Release Notes
+
+This release includes link:https://access.redhat.com/errata/RHSA-2025:3907[RHSA-2025:3907].
+
+[id="logging-release-notes-6-1-5-enhancements_{context}"]
+== New features and enhancements
+
+* Before this update, time-based stream sharding was not enabled in Loki, which resulted in Loki being unable to save historical data. With this update, {loki-op} enables time-based stream sharding in Loki, which helps Loki save historical data. (link:https://issues.redhat.com/browse/LOG-6991[LOG-6991])
+
+
+
+
+[id="logging-release-notes-6-1-5-bug-fixes_{context}"]
+== Bug fixes
+
+* Before this update, the Vector collector could not forward Open Virtual Network (OVN) and Auditd logs. With this update, the Vector collector can forward OVN and Auditd logs. (link:https://issues.redhat.com/browse/LOG-6996[LOG-6996])
+
+[id="logging-release-notes-6-1-5-cves_{context}"]
+== CVEs
+
+* link:https://access.redhat.com/security/cve/CVE-2022-49043[CVE-2022-49043]
```

Comment:
I only see one CVE https://access.redhat.com/security/cve/CVE-2025-30204. Can you update the list?

Reply:
thanks, done